Source/WebKit/UIProcess/wpe/WebPageProxyWPE.cpp

```diff
@@ -192,6 +192,18 @@ OptionSet<WebCore::PlatformEvent::Modifier> WebPageProxy::currentStateOfModifier
 #endif
 }
 
+bool WebPageProxy::useExplicitSync() const
+{
+#if ENABLE(WPE_PLATFORM)
+    auto* view = wpeView();
+    if (!view)
+        return false;
+    return wpe_display_use_explicit_sync(wpe_view_get_display(view));
+#else
+    return false;
+#endif
```

Comment:
With previous suggestion we don't need the `#else` just `#endif` and return false